packages/react-instantsearch-core-next/src/useStableValue.ts

```diff
@@ -0,0 +1,15 @@
+import { dequal } from 'dequal/lite';
+import { useEffect, useState } from 'react';
+
+export function useStableValue<TValue>(value: TValue) {
```

Comment:
We use the same implementation as in Recommend UI: https://github.com/algolia/recommend/blob/b3195f9c07d09a3f97b9ca980cfadedafd3a9913/packages/recommend-react/src/useStableValue.ts

Reply:
I'm preparing a test for it still.